.travis.yml

```diff
@@ -97,7 +97,7 @@ install:
 script:
   - python labcontrol/gui/js_tests/js_tester.py all-tests
   - nosetests --exclude-dir=labcontrol/gui/js_tests --with-doctest --with-coverage -v --cover-package=labcontrol
-  - flake8 labman setup.py scripts/*
+  - flake8 labcontrol setup.py scripts/* --max-line-length 120
```

Comment:
Sorry--The 80-character line length limit (the `flake8` default) is standard in Knight Lab software (e.g., Qiita, deblur, EMPeror, biom-format, etc), so we really can't drop it unilaterally. Pls regress that change.

Reply:
My impression was this is the case because 79-characters is PEP8. The main reason for abiding PEP8 being readability and stylistic consistency. In cases such as this, I am tempted to invoke [this](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds) part of PEP8.  In my opinion, aggressively line-splitting in the case of some of these SQL queries can reduce readability.

That being said, if you insist, I am happy to oblige.

Reply:
I tend to agree with you about line-splitting the SQL statements reducing readability!  However, my understanding (correct me if I'm wrong!) is that setting the max line length of flake8 to 120 will mean it will not alert on ANY line in the code that is <120 lines long, including the many that are pure python and should stay at the 80-character limit for consistency with other projects.  You could, though, mark of the SQL statements that you want to have longer lines with `# noqa: E501` (see http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors ), which would prevent them from causing flake8 warnings while still letting the rest of the codebase follow the shared convention.

Reply:
> setting the max line length of flake8 to 120 will mean it will not alert on ANY line in the code that is <120 lines long, including the many that are pure python and should stay at the 80-character limit for consistency with other projects.
I believe this is correct as well.
>  You could, though, mark off the SQL statements that you want to have longer lines with # noqa: E501 (see http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors ), which would prevent them from causing flake8 length warnings while still letting the rest of the codebase follow the shared convention.
Oh that's cool, I haven't seen that before, thanks! 
That being said, I think in this instance, the `# noqa: E501` solution is less than 'developer-friendly' solution if these lines ever need to be updated, and for future instances of _a little too long for your average python line_ SQL queries. Unless your line lengths are setting off red-flags for the reviewer, I feel that hard limit at 120, self-enforced soft limit at 79 is a reasonably provident solution. Feel free to disagree, we can chat at the meeting later :)

Reply:
I talked with @AmandaBirmingham and we were not sure if we have the authority to increase the python line-length limit for this project. @antgonza or @ElDeveloper can either of you comment on whether this seems reasonable?